documentation/docs/pipelines/cloud-sdk/configuration.md

```diff
@@ -462,34 +462,25 @@ The options for the use of linting tools remain the same and are described in th
 
 Note, the available configuration options can be found in the related [step documentation](https://sap.github.io/jenkins-library/steps/npmExecuteLint/#parameters).
 
-### sonarQubeScan
+### compliance
```

Comment:
in the long term we should probably get rid of those sub-sections and move content into the stage docs itself, I guess. should be fine for now to leave it here